data/src/test/java/org/apache/iceberg/TestSplitScan.java

```diff
@@ -63,7 +63,7 @@
 
   @Parameterized.Parameters(name = "format = {0}")
   public static Object[] parameters() {
-    return new Object[] { "parquet", "avro" };
+    return new Object[] {"parquet", "avro"};
```

Comment:
Nit: this line doesn't need to change. Can you revert it to avoid commit conflicts?